apps/docs/sway/script-simple-log/src/main.sw

```diff
@@ -0,0 +1,10 @@
+// #region full
+script;
+
+use std::logging::log;
```

Comment:
I am almost certain that there is no need to explicitly import the log fn anymore